client/src/components/SaladListComponent.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react"
+import SaladTile from "./SaladTile"
+
+
+const SaladListComponent = (props) => {
+
+    const [salads, setSalads] = useState([])
+
+    const getSalads = async() =>{
+        try{
+            const response = await fetch("/api/v1/salads")
+            if(!response.ok){
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const responseBody = await response.json()
+            setSalads(responseBody.salads)
+        }catch(error){
+            console.error(`Error in Fetch: ${error.message}`)
+        }
+    }
+
+    useEffect(()=> {
+        getSalads()
+    }, [])
+
+    const saladItems = salads.map((salad)=> {
+        
+        return (
+            <SaladTile
+            key={salad.id}
+            salad={salad}
+            />
+        )
+    })
+
+    return (
+        <div>
+        <h1>My Salad List</h1>
+        <ul>{saladItems}</ul>
```

Comment:
the `h1` and `ul` should be indented as they are children of the `div`

consider changing the `ul` to a `div`, as an `ul` expects `li` elements as its children, but the `saladItems` are a collection of `div` elements